Source/Menu/Options.cs

```diff
@@ -538,6 +539,15 @@ void buttonOK_Click(object sender, EventArgs e)
             Settings.Save();
         }
 
+        private string GetValidWindowSize()
+        {
+            // "1024X780" instead of "1024x780" then "Start" resulted in an immediate return to the Menu with no OpenRailsLog.txt and a baffled user.
+            var windowSizeArray = comboWindowSize.Text.ToLower().Replace(" ", "").Split('x');
+            return (int.TryParse(windowSizeArray[0], out int width) && int.TryParse(windowSizeArray[1], out int height))
```

Comment:
this will crash if the string.Split does not return an array of expect length, and should be checked also!